### ISSUE TITLE
✨ Admin / Classifications / mostly Styling update

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#11427]
## ✨ Feature

Update styling of the Community pages, standardize the way things work, and bring it in line with the overall platform's look and feel.

### Out of scope for this issue (don't build this even if it's in the design)
- [ ] Moving to the new nav
- [ ] Updates to the table component
- [ ] Adding "searchable" to the classification model (and associated UI)
- [ ] Advanced (or advanced tools) tab
- [ ] Changes to table search and filter

## 🎨 Design

[Figma file](https://www.figma.com/design/Ov3ZwbSJXh7Ckqk6cefPDo/Admin---Classifications?node-id=5437-34104&t=NdEom57TwzcRXijs-1) by @Jerryescandon 

## ✅ Acceptance Criteria

- [ ] Classifications page (/admin/settings/classifications)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Remove "edit" column, add edit link to the "Name" column instead
- [ ] Create classification (/admin/settings/classifications/create)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
- [ ] Edit a community page (/admin/classifications/:id:/edit)
  - [ ] (should use same component as Create Classification page)
- [ ] New page - Classifications information (/admin/classifications/:id:)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Only one tab for now
  - [ ] Inline form for displaying information and editing

- [ ] Mobile view for all pages

- [ ] Design Review by @Jerryescandon 

## Blockers
```[tasklist]
### Blocked by
- [ ] #11344 
```

**Comment by gobyrne** [comment]
This one also ready for your review @NienkeBr  (no rush)

**Comment by gobyrne** [comment]
@Jerryescandon maybe only let people select "group & level" during the Create step. Actually, let's think about it what fields should be "not editable" after creation.

Then we have the design question about how to show them during create step. But also, seing the on the View page, but not the edit page?

**Comment by gobyrne** [comment]
@Jerryescandon *create page is missing level & salary. Maybe clean up the "display names" too.

**Comment by Jerryescandon** [comment]
Updated screenshots:
- Added group and level and salary to create page
- Removed display name fields
